### DISCUSSION TITLE
现在版本没有自动调整分辨率了吗？

### DISCUSSION THREAD

**Discussion by 16cm** [the1812/Bilibili-Evolved#1661] [提问 / Q&A] [answered]
现在版本没有自动调整分辨率了吗？

Bilibili Evolved：1.12.0
Edge：89.0.774.54

**Comment by the1812** [comment, marked as answer]
早没了

视频本来就能记住
直播目前没办法自动改

**Reply by 16cm** [reply]
哦哦，感谢~

**Comment by Geeyun-JY3** [comment]
文档里有说明：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features.md#视频
